src/Application/Drive/View/Sidebar.elm

```diff
@@ -523,3 +560,34 @@ detailsForSelection { showPreviewOverlay } sidebar model =
             |> Maybe.withDefault
                 nothing
         ]
+
+
+
+-- UTILITIES
+
+
+onCtrlS : msg -> Html.Attribute msg
+onCtrlS message =
+    let
+        ensureEquals value decoder =
+            decoder
+                |> D.andThen
+                    (\val ->
+                        if val == value then
+                            D.succeed ()
+
+                        else
+                            D.fail "Unexpecated value"
+                    )
+    in
+    E.custom "keydown"
```

Comment:
I only tend to use the "single letter module" pattern for attributes, can you use a more elaborate name here? In Elm, it is possible to import the same module multiple times with different names, right? 🤔 

Same for `D.`

Reply:
`E` was already imported this way in this module. It's for `Html.Attribute.Events.custom`. I guess from just seeing `E.custom` it's not obvious that this is a custom event handler, but I'd keep it this way to keep consistency.

I've changed it for `D.` though. :+1: